x-pack/plugins/cases/common/ui/types.ts

```diff
@@ -26,15 +26,6 @@ export interface CasesContextFeatures {
 
 export type CasesFeatures = Partial<CasesContextFeatures>;
 
-export interface CasesContextValue {
```

Comment:
this was moved out of here as it is not a common type for BE and FE but only FE.

Reply:
@academo Out of curiosity what does BE and FE stand for?

Reply:
BE = Backend, FE = Frontend.